types/index.d.ts

```diff
@@ -13,20 +13,26 @@ export interface ChalkOptions {
 	level?: Level;
 }
 
-export interface Chalk {
+export interface ChalkConstructor extends Chalk {
```

Comment:
Why not use classes rather than Interfaces when the Object is clearly a class here?